server/runtime.go

```diff
@@ -716,6 +723,10 @@ func NewRuntime(logger, startupLogger *zap.Logger, db *sql.DB, jsonpbMarshaler *
 		allBeforeReqFunctions.beforeAuthenticateFacebookFunction = goBeforeReqFunctions.beforeAuthenticateFacebookFunction
 		startupLogger.Info("Registered Go runtime Before function invocation", zap.String("id", "authenticatefacebook"))
 	}
+	if goBeforeReqFunctions.beforeAuthenticateFacebookInstantGameFunction != nil {
```

Comment:
Also add support for Lua before hook, see: https://github.com/heroiclabs/nakama/blob/master/server/runtime_lua.go#L229

Reply:
Done